extensions/crypto/Cargo.toml

```
@@ -15,4 +15,6 @@ path = "lib.rs"
 
 [dependencies]
 deno_core = { version = "0.88.1", path = "../../core" }
+tokio = { version = "1.4.0", features = ["full"] }
```

```suggestion
tokio = { version = "1.6.1", features = ["full"] }
```
(That's what the other crates currently use.)